src/System.Private.CoreLib/shared/System/Activator.RuntimeType.cs

```diff
@@ -132,26 +132,122 @@ internal static object CreateInstance(Type type, bool nonPublic, bool wrapExcept
             }
 
             if (type == null)
-            {                
+            {
                 type = assembly.GetType(typeName, throwOnError: true, ignoreCase);
             }
 
             object o = CreateInstance(type, bindingAttr, binder, args, culture, activationAttributes);
 
-            return o != null ? new ObjectHandle(o) : null;          
+            return o != null ? new ObjectHandle(o) : null;
         }
 
-        public static T CreateInstance<T>()
-        {
-            var rt = (RuntimeType)typeof(T);
+        public static T CreateInstance<T>() => CreateInstance<T>(publicOnly: true, wrapExceptions: true);
 
+        private static T CreateInstance<T>(bool publicOnly, bool wrapExceptions)
```

Comment:
Can this method be used in `CreateInstance(Type type, bool nonPublic, bool wrapExceptions)` without dropping AOT support?